src/transport/reqwest.rs

```diff
@@ -257,9 +266,43 @@ impl PubNubClientBuilder<TransportReqwest> {
     /// [`TransportReqwest`]: ./struct.TransportReqwest.html
     /// [`reqwest`]: https://docs.rs/reqwest
     /// [`PubNub API`]: https://www.pubnub.com/docs
-    pub fn with_reqwest_transport() -> PubNubClientBuilder<TransportReqwest> {
-        PubNubClientBuilder {
-            transport: Some(TransportReqwest::new()),
+    #[cfg(all(any(feature = "subscribe", feature = "presence"), feature = "std"))]
```

Comment:
I think we should @techwritermat to make him aware about that small details about our builders. 

(tyi: depending of feature set - we return different builders that allows users to set needed stuff without letting them creating runtime errors. All miss-configurations will be found in the compile time).